junior/app/styles/components/challenge/challenge-content.scss

```diff
@@ -1,43 +1,60 @@
 .challenge-content {
-  display: flex;
-  gap: var(--pix-spacing-6x);
+  display: grid;
+  grid-row-gap: var(--pix-spacing-4x);
+  grid-column-gap: var(--pix-spacing-6x) ;
+  grid-template-columns: 1fr;
   margin: 16px 140px 0;
   transition: all 0.4s;
 
-  @include device-is('tablet') {
-    margin: 16px 120px 0;
+  iframe {
+    min-height: 100%;
   }
 
-  &--single-display {
-    flex-direction: column;
+  .challenge-embed-simulator,
+  .challenge-media__loaded-image,
+  .challenge-media__placeholder{
+    padding: var(--pix-spacing-4x);
+    background-color: var(--pix-neutral-0);
+    border-radius: 24px;
+    box-shadow: 0 0 1px 0 var(--pix-primary-500), 8px 8px 32px -8px rgb(63 125 216 / 35%)
   }
 
-  &__media,
-  &__proposals {
-    flex: 1;
-  }
 
-  &__media {
-    &--framed {
-      padding: var(--pix-spacing-4x);
-      background-color: var(--pix-neutral-0);
-      border-radius: 24px;
-      box-shadow: 0 0 1px 0 var(--pix-primary-500), 8px 8px 32px -8px rgb(63 125 216 / 35%);
+  &__grid-multiple-element {
+    grid-template-areas:
+    "illu  form"
+    "illu actions";
+    grid-template-columns: 1fr 1fr;
+
+    &--1x-2x {
+      grid-template-columns: 40% 60%;
+    }
+
+    div:nth-child(1) {
+      grid-area: illu;
+    }
+
+    div:nth-child(2) {
+      grid-area: form;
+    }
+
+    div:nth-child(3) {
+      grid-area: actions;
     }
   }
 
-  &__qrocm {
-    align-items: center;
+  @include device-is('tablet') {
+    margin: 16px 120px 0;
   }
 
-  &__qcm, &__qcu {
+  &__questionnaire {
     display: flex;
     justify-content: center;
     width: 100%;
   }
 
   &__autoreply {
-    width: 0;
+    display: none;
```

Comment:
Vérifier que les embed auto-reply fonctionnent toujours.

Reply:
it is OK!!!!!!!